example/use_cases/employer_university_flow/pkg/verifier.go

```diff
@@ -0,0 +1,54 @@
+package pkg
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/signing"
+	"github.com/TBD54566975/ssi-sdk/cryptosuite"
+	"github.com/sirupsen/logrus"
+)
+
+// This is a very simple validation process.
+// against a Presentation Submission
+// It checks:
+// 1. That the VC is valid
+// 2. That the VC was issued by a trusted entity
+func validateAccess(verifier cryptosuite.JSONWebKeyVerifier, data []byte) error {
+
+	var authorizationError = errors.New("insufficient claims provided")
```

Comment:
either raise to be outside the method or `authorizationError := errors.New(....`